### PR TITLE
Problems

### PR DESCRIPTION
**Pull request by VinnyLuu**
Added problem deletion functionality. Made a popup window for the activity. But there is again the problem of the problem list being updated like when a problem is added and then returning to the problem list

### CODE REVIEW

**Comment by itstc** [comment]
The problemlist not updating could be one of the following:
* In render, you are not updating the ListAdapter holding the problems. The solution to this would be setting the ListAdapters current list to the newest state and invoking notifyDataSetChanged() in render.
* The server is taking too long to delete the problem, thus we are querying the problems before the problem is deleted. A solution to this is to sleep the DeleteProblemTask after it executes the Delete action to the database.

**Comment by VinnyLuu** [comment]
screen shot 2018-11-18 at 2 31 54 pm
screen shot 2018-11-18 at 2 32 10 pm
screen shot 2018-11-18 at 2 32 25 pm
screen shot 2018-11-18 at 2 33 05 pm

File: app/src/main/java/com/team7/cmput301/android/theirisproject/controller/AddProblemController.java [hunk 1]
```diff
@@ -17,7 +17,11 @@
 import com.team7.cmput301.android.theirisproject.model.BodyPhoto;
 import com.team7.cmput301.android.theirisproject.task.AddProblemTask;
 
+<<<<<<< HEAD
```
Comment (Jmmxp) [inline app/src/main/java/com/team7/cmput301/android/theirisproject/controller/AddProblemController.java]: Didn't fix merge conflict here

File: app/src/main/java/com/team7/cmput301/android/theirisproject/controller/DeleteProblemController.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+/**
+ * DeleteProblemController has methods to allow DeleteProblemActivity
+ * to interact with the database to delete problems.
+ *
+ * @author VinnyLuu
+ * @see DeleteProblemTask
+ * */
+public class DeleteProblemController extends IrisController {
+    String problemID;
```
Comment (Jmmxp) [inline app/src/main/java/com/team7/cmput301/android/theirisproject/controller/DeleteProblemController.java]: Should be private

File: app/src/main/java/com/team7/cmput301/android/theirisproject/model/Problem.java [hunk 3]
```diff
@@ -4,7 +4,10 @@
 
 package com.team7.cmput301.android.theirisproject.model;
 
+<<<<<<< HEAD
```
Comment (Jmmxp) [inline app/src/main/java/com/team7/cmput301/android/theirisproject/model/Problem.java]: Merge conflict

File: app/src/main/java/com/team7/cmput301/android/theirisproject/task/DeleteProblemTask.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+            if (res.isSucceeded()) {
+                return true;
+            }
+            else {
+                return false;
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return null;
```
Comment (Jmmxp) [inline app/src/main/java/com/team7/cmput301/android/theirisproject/task/DeleteProblemTask.java]: Should be returning false here at the end if anything, then you can take out the else branch above

File: app/src/main/res/layout/activity_add_problem.xml [hunk 5]
```diff
@@ -27,6 +27,18 @@
                 android:maxLength="30"
                 app:layout_constraintTop_toBottomOf="@+id/problem_title_edit_text" />
 
+<<<<<<< HEAD
```
Comment (Jmmxp) [inline app/src/main/res/layout/activity_add_problem.xml]: Merge conflict

File: app/src/main/res/layout/activity_problem_list.xml [hunk 6]
```diff
@@ -12,6 +12,13 @@
     android:layout_height="match_parent"
     tools:context=".activity.ProblemListActivity">
 
+<<<<<<< HEAD
```
Comment (Jmmxp) [inline app/src/main/res/layout/activity_problem_list.xml]: Merge conflict

**Pull request merged**